### PR TITLE
a cruel idea

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl:
fix: Adds Reddit to the game.
/:cl:

is this a good idea y/n

### CODE REVIEW

**Comment by Core0verload** [comment]
It's a horrible, horrible idea. Instant :+1:

File: code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -28,11 +28,11 @@
 			usr << "<span class='danger'>OOC for dead mobs has been turned off.</span>"
 			return
 		if(prefs.muted & MUTE_OOC)
```
Comment (Core0verload) [inline code/modules/client/verbs/ooc.dm]: Unnecessary I think. This is not for ban, it's for OOC turned off by player in options.

**Comment by Firecage** [comment]
Shadowbans are a retarded idea honestly. If someone is muted or banned from OOC, they should be informed of it when they try to use it. They shouldn't be thrown for a fucking loop.

**Comment by Jordie0608** [comment]
I forgot to close this instantly.

File: code/modules/client/verbs/ooc.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/client/verbs/ooc.dm]: nono, thats OOC temp mutes, check above this

**Pull request closed without merging**